Apps/Src/FaultState.c

```diff
@@ -145,40 +107,34 @@ void EnterFaultState() {
 #ifdef DEBUGMODE
     char command[COMMAND_SIZE];
 #endif
-    BSP_WDTimer_Init(); //This is in case we did not pass check contactor and watchdog timer was not initialized
+    BSP_WDTimer_Init(true); //This is in case we did not pass check contactor and watchdog timer was not initialized
```

Comment:
comment should be changed to "change watchdog reload value to max to for fault state" cause task_check_contactor and task_watchdog are both initialized by task_init and watchdog has a lower prio# so i p sure its initialized before check_contactor can run

Reply:
u right